build.sc

```diff
@@ -25,18 +26,19 @@ import $file.example.websockets2.build
 import $file.example.websockets3.build
 import $file.example.websockets4.build
 import $file.ci.upload
-import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.1.4`
+import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.3.0`
 import de.tobiasroeser.mill.vcs.version.VcsVersion
 
-val scala213 = "2.13.8"
-val scala212 = "2.12.16"
-val scala3 = "3.1.3"
+val scala213 = "2.13.10"
+val scala212 = "2.12.17"
+val scala3 = "3.2.2"
```

Comment:
Is the upgrade to Scala 3 necessary because of dependencies? If not, I would suggest that if it *can* be kept at a lower version, that we should be conservative. Scala 3.x is only backwards compatible and not forwards compatible, so I always recommend that libraries used by other projects try to stay on the lowest possible version

Reply:
I assume he bumped it because i bumped it in upickle, and i bumped it in upickle because i was getting weird macro errors that went away when i upgraded

Reply:
ok, that's  a pretty convincing justification